#*H
#  Copyright 2017 Transport Systems Catapult (The Catapult)
#  All rights reserved
#  For use only for the purpose agreed with The Catapult
#H*

Header header
PodHeader pod_header
bool output0
bool output1
bool output2
bool output3
bool output4
bool output5
bool output6
bool output7
